numpy/core/tests/test_overrides.py

```diff
@@ -147,6 +147,19 @@ def test_too_many_duck_arrays(self):
         with assert_raises_regex(TypeError, 'distinct argument types'):
             _get_implementing_args(relevant_args)
 
+    def test_where_duck_array(self):
+
+        class DuckArray:
+            def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
+                return "overridden"
+
+        array = np.array(1)
+        duck_array = DuckArray()
+
+        result = np.negative(array, where=duck_array)
```

Comment:
What do we think about only testing `numpy.negative` here? Is that sufficient or is there a better way to test "all" ufuncs.

Reply:
I think it is good enough, though perhaps it is a good idea to verify that its override is considered last? I.e., put in another override in either input or output and check that those have precedence...